TomorrowlandSessionPlanner/TomorrowlandSessionPlanner/TomorrowlandSessionPlanner.csproj

```diff
@@ -13,7 +13,7 @@
         <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
       </PackageReference>
       <PackageReference Include="Microsoft.EntityFrameworkCore.Sqlite" Version="8.0.1" />
-      <PackageReference Include="MudBlazor" Version="6.15.0" />
+      <PackageReference Include="MudBlazor" Version="7.0.0" />
```

Comment:
For this, MudBlazor has to bee adjusted before merging this into main or develop (Breaking Changes).